site/source/user-manual/service-guides/lightning/lightning-integrations.rst

```diff
@@ -86,7 +86,145 @@ Make sure you are already :ref:`running Tor<connecting-tor>` on your system and
 BlueWallet
 ----------
 
-.. note:: Not available at this time - LNDHub is required
+.. note:: Not available at this time.
+
+.. _lnbits-zeus:
+
+LNbits & Zeus
+-------------
+
+.. note:: LND only
+
+.. note:: Requires Tor to be running on your phone - use :ref:`this guide<tor-os>` if necessary.
+
+Using LnBits allows you to create a wallet that makes use of your node with only an alloted amount of sats. This restriction can be very helpful for if you only want to have a small amount for spending on your phone without making your entire lightning balance available. You can even allow other people to have wallets you create for them - think giving your children an allowance. They can start with a set amount (can be zero) and simply spend what you initially make available. They can also earn more for themselves as they will have the freedom to issue their own invoices - all while making use of your node.
+
+.. tip:: All wallets created this way are ultimately bound by the capacity of your node. If one wallet is allocated 10,000 sats but your underlying node only has 9000 sats of outbound capacity, payments will simply fail.
+
+#. Start by ensuring that you have LND and LNbits installed already. You also need for LND to have at least one channel set up otherwise payments will not work. If you have not set up a channel yet, please follow :ref:`this guide<lightning-intro>`.
+
+   .. figure:: /_static/images/services/lnbits/lnbits-lnd-installed.png
+      :width: 40%
+      :alt: lnbits-lnd-installed
+
+#. Head to LNbits on your server, click on **Properties** and copy address under the **Superuser Account**.
+
+   .. note:: You need to make sure you are going to .onion address - Zeus cannot connect over .local! Please change *.local* to *.onion* and remove the *s* from *https://* as shown in the picture below.
+
+   .. figure:: /_static/images/services/lnbits/local-to-onion.png
+      :width: 60%
+      :alt: local-to-onion
+
+#. Now a default wallet will be generated - highlighted on the top left. We'll rename it by clicking **Rename wallet** entering **android-zeus-wallet** then clicking **UPDATE NAME**:
```

Comment:
"Now a default wallet will be generated"? After changing the URL to http and .onion? How did you get to this screen? The last thing you told me to do was go into properties and chnage the URL. Now suddenly you are in some screen creating a wallet.